modal/modal.css

```diff
@@ -0,0 +1,39 @@
+#modal-content{
+    position: fixed; 
+    z-index: 1; 
+    left: 0;
+    top: 12;
+    width: fit-content;
+    height:fit-content; 
+    background-color:white;
+    border-radius: 5px;
+    margin: 1rem;
+    border: 1px solid whitesmoke;
+}
+
+.modal-body{
+    padding: 1rem;
+    gap: 1rem;
+}
+
+.space-btwn{
+    justify-content: space-between; 
+}
+
+.modal-btn-div{
+    padding: 10px;
+    display: flex;
+    justify-content: flex-end;
+    column-gap: 1rem;
+    background-color: rgb(226, 222, 216);
```

Comment:
if this color is being used multiple times, use css var

Reply:
yeah that color is used only once hence not used css var